lib/cli.ex

```diff
@@ -4,18 +4,15 @@ defmodule Onigumo.CLI do
   }
 
   def main(argv) do
-    case OptionParser.parse(
-           argv,
-           aliases: [C: :working_dir],
-           strict: [working_dir: :string]
-         ) do
-      {parsed_switches, [component], []} ->
-        {:ok, module} = Map.fetch(@components, String.to_atom(component))
-        working_dir = Keyword.get(parsed_switches, :working_dir, File.cwd!())
-        module.main(working_dir)
+    parse_result =
```

Comment:
`parse` should be enough, but at least `parsed_result` with **d** is more semantically right in my opinion.

Reply:
🤔  `parsed_result` doesn’t seem right to me: it’s not a result that has been parsed; it’s a result of parsing. `parse_result` tried to convey the message that it’s a result of the `parse` function.

Shortened to a single word, `parsed` may be a better choice. `parse` sounds like an action rather than a value.